hw/ip/sysrst_ctrl/data/sysrst_ctrl_testplan.hjson

```diff
@@ -203,29 +203,25 @@
     {
```

Comment:
Can you change the tag of your commit to be `[hw,sysrst_ctrl]`?

Reply:
updated commit message